### DISCUSSION TITLE
Where to get installation output key if my laptop crashed?

### DISCUSSION THREAD

**Discussion by BossyBigBoss** [outline/outline#7578] [General]
I've had Outline Manager on my crashed laptop.
I did not save the key to access the Outline server.  

Where to get it to have access again?
I have a lot of users on the server and don't want to reinstall it from scratch.

**Comment by tommoor** [comment]
You're looking for https://github.com/Jigsaw-Code/outline-apps

**Reply by BossyBigBoss** [reply]
I don't need Outline Manager itself.
I know where to download it.

I need a code that I need to paste to get access to my Outline server.
The code that was generated first time during the installation.

I did not save at and just pasted it to the window in the Outline Manager.

How to get it again?
